parkhangs-webapp/src/router/Routes.js

```diff
@@ -7,22 +7,24 @@ import NoMatch from 'views/NoMatch'
 import Modal from 'features/modal/Modal'
 import 'App.css'
 import { connect } from "react-redux";
+import UserInfo from "../components/UserInfo";
```

Comment:
would prefer if you move this file into the views folder, since it's a main view rather than a shared component